generated/nidaqmx/constants.py

```diff
@@ -525,12 +534,12 @@ class PowerUpChannelType(Enum):
 class PowerUpStates(Enum):
     HIGH = 10192  #: Logic high.
     LOW = 10214  #: Logic low.
-    TRISTATE = 10310  #: High-impedance state. You can select this state only on devices with bidirectional lines.  You cannot select this state for dedicated digital output lines. On some devices, you can select this value only for entire ports.
+    TRISTATE = 10310  #: High-impedance state. You can select this state only on devices with bidirectional lines. You cannot select this state for dedicated digital output lines. On some devices, you can select this value only for entire ports.
 
 
 class PressureUnits(Enum):
     PASCALS = 10081  #: Pascals.
-    POUNDS_PER_SQ_INCH = 15879  #: Pounds per square inch.
+    POUNDS_PER_SQUARE_INCH = 15879  #: Pounds per square inch.
```

Comment:
Good!

Reply:
`PressureUnits.POUNDS_PER_SQ_INCH` is used in nidaqm-python repo. Thus have retained the expected enum value name (`POUNDS_PER_SQ_INCH`)